src/common.ts

```diff
@@ -197,11 +197,18 @@ export const ardriveOracle = new ArDriveContractOracle([
 export async function validateTxTip(node: GQLNodeInterface, ardriveOracle: ArDriveContractOracle): Promise<boolean> {
 	const tags = node.tags;
 	const boostValue = +(tags.find((tag) => tag.name === BOOST_TAG)?.value || '1');
-	const appName = tags.find((tag) => tag.name === APP_NAME_TAG)?.value;
-	const appVersion = tags.find((tag) => tag.name === APP_VERSION_TAG)?.value;
+	const appNameTags = tags.filter((tag) => tag.name === APP_NAME_TAG);
+	const appVersionTags = tags.filter((tag) => tag.name === APP_VERSION_TAG);
 	const bundledIn = node.bundledIn;
+
 	const isV2Tx = !bundledIn;
-	if (appName === WEB_APP_NAME && appVersion && !isSemanticVersionGreaterThan(appVersion, '1.14.1') && isV2Tx) {
+	const isWebApp = appNameTags.some((appNameTag) => appNameTag.value === WEB_APP_NAME);
+	const isAppVersionGreaterThan1_14_1 = appVersionTags.some((appVersionTag) =>
+		// FIXME: this way of doing it cannot distinguish between multiple apps' version
+		// might result in false positives
+		isSemanticVersionGreaterThan(appVersionTag.value, '1.14.1')
+	);
+	if (isWebApp && !isAppVersionGreaterThan1_14_1 && isV2Tx) {
```

Comment:
Should we have an array of ArDrive versions somewhere just to be more precise? Hard to track?

Reply:
Hard to track and even with that there's a possibility of other apps using App-Version to have the same versions that we want.

Reply:
Just provide a comment here saying that until SmartWeaveAction reaches v1.14.1 we're good

Reply:
There it is - a732b61245cd55f6bc21925a1c7adb0e2d0f7c09